latest/ug/automode/create-node-class.adoc

```diff
@@ -175,9 +175,12 @@ spec:
     # Key Alias Name
     # Key Alias ARN
   
-  # Optional: Forward proxy, commonly requires certificateBundles as well
-  #for EC2, see https://repost.aws/knowledge-center/eks-http-proxy-containerd-automation
   advancedNetworking:
+    # Optional: Controls whether public IP addresses are assigned to instances that are launched with the nodeclass.
```

Comment:
Should we add a note that, if unspecified, this is dictated by the subnet settings?